cmd/ike/watch/watch_test.go

```diff
@@ -147,10 +147,10 @@ src/main/**/*.java
 		code := TmpFile(GinkgoT(), watchTmpDir+"/main.go", "package main")
 
 		defer func() {
-			config.Close()
-			test.Close()
-			nestedFile.Close()
-			gitIgnore.Close()
+			_ = config.Close()
```

Comment:
print error instead of just completely ignore?

Reply:
That would be an improvement :P